core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -716,6 +716,11 @@ void importDirectory(String tableName, String dir, String failureDir, boolean se
      */
     ImportMappingOptions tableTime(boolean value);
 
+    /**
+     * Ignores empty bulk import source directory, rather than throwing an IllegalArgumentException.
+     */
+    ImportMappingOptions ignoreEmptyDir(boolean ignore);
```

Comment:
Should include `@since 2.1.0` java doc tag since this will be new.

Reply:
Tag was added.